htdocs/core/class/dolgraph.class.php

```diff
@@ -1280,21 +1338,16 @@ private function draw_chart($file, $fileurl)
 				$this->stringtoshow .= 'label: \'' . dol_escape_js(dol_string_nohtmltag($textoflegend)) . '\', ';
 				$this->stringtoshow .= 'pointStyle: \'' . ((!empty($this->type[$i]) && $this->type[$i] == 'linesnopoint') ? 'line' : 'circle') . '\', ';
 				$this->stringtoshow .= 'fill: ' . ($type == 'bar' ? 'true' : 'false') . ', ';
-				if ($isfunnel) {
-					$this->stringtoshow .= 'borderWidth: \'2\', ';
-				} elseif ($type == 'bar' || $type == 'horizontalBar') {
-					$this->stringtoshow .= 'borderWidth: \'1\', ';
+				if ($type == 'bar' || $type == 'horizontalBar') {
+					$this->stringtoshow .= 'borderWidth: \''.$this->borderwidth.'\', ';
 				}
 				$this->stringtoshow .= 'borderColor: \'' . $bordercolor . '\', ';
 				$this->stringtoshow .= 'backgroundColor: \'' . $color . '\', ';
-				if (!empty($arrayofgroupslegend) && !empty($arrayofgroupslegend[$i])) $this->stringtoshow .= 'stack: \'' . $arrayofgroupslegend[$i]['stacknum'] . '\', ';
-				$this->stringtoshow .='data: [';
-				if ($isfunnel) {
-					$this->stringtoshow .= '['.-$serie[$i].','.$serie[$i].']';
-				} else {
-					$this->stringtoshow .= $serie[$i];
-				}
-				$this->stringtoshow .=']';
+				if ($arrayofgroupslegend[$i]) $this->stringtoshow .= 'stack: \'' . $arrayofgroupslegend[$i]['stacknum'] . '\', ';
```

Comment:
The previous code had this test:
if (!empty($arrayofgroupslegend) && !empty($arrayofgroupslegend[$i]))
It should be same than 
if ($arrayofgroupslegend[$i]) 
but will avoid warning if $arrayofgroupslegend is not defined

Does this removal was done on purpose ?

Reply:
Sorry it was a miss remove